node.gyp

```diff
@@ -985,7 +986,13 @@
               'ldflags': [
                 '-Wl,-blibpath:/usr/lib:/lib:/opt/freeware/lib/pthread'
               ],
-            }]
+            }],
+            ['"<(real_os_name)"=="OS400"', {
+              'ldflags': [
+                '-Wl,-blibpath:/QOpenSys/pkgs/lib:/QOpenSys/usr/lib',
+                '-Wl,-bbigtoc', '-Wl,-brtl'
```

Comment:
Can you put the second argument on its own line?  And maybe add a comma to keep future diffs smaller, same on lines 104 and 369.